src/Schema.php

```diff
@@ -90,6 +93,17 @@ public static function ensureFrontControllerAccess() {
     static::createOrPrependFile(ABSPATH . '.htaccess', $template, "\n");
   }
 
+  /**
+   * Set  Expires + Cache-Control header with a TTL of 1 year for all assets.
+   */
+  public static function setAssetsExpiryDate() {
+    $hostname = parse_url(site_url())['host'];
+    if (strpos($hostname, 'nest') === FALSE && strpos($hostname, 'local') === FALSE && !WP_DEBUG) {
```

Comment:
The exclusion cannot happen at installation time - it needs to happen at runtime, regardless of where the plugin was installed.